src/main/java/redis/clients/jedis/commands/AdvancedBinaryJedisCommands.java

```diff
@@ -42,4 +42,9 @@
   String clientSetname(byte[] name);
 
   byte[] memoryDoctorBinary();
+  
+  byte[] memoryUsageBinary(byte[] key);
+
+  byte[] memoryUsageBinary(byte[] key, int samples);
```

Comment:
`Binary` is not required because of presence of `byte[]`.

Reply:
@sazzad16 yes I know, but I noticed other methods are keeping this patter so I followed it

Reply:
done